### ISSUE TITLE
Wrong font color on numbered list when using dark theme

### ISSUE THREAD

**Issue by hanahaneull** [CarlosEsco/Neko#1585]
### Steps to reproduce

1. Use any dark theme
2. See manga description that have numbered list description [here](https://mangadex.org/title/14127169-0b98-49f4-b495-7fdabf803f54/tawawa-na-oppai-wa-suki-desu-ka-kyonyuu-shoujo-anthology-comic) (NSFW)

### Expected behavior

Font color for numbered list should match with dark theme settings

### Actual behavior

Numbered font color same as light theme

### Neko version

2.14.0

### Android version

Android 11

### Device

Infinix X6812B

### Other details

![Screenshot_20231031-025849](https://github.com/CarlosEsco/Neko/assets/48522963/018d841e-9af0-42f5-b57f-f88eb9e206af)
![Screenshot_20231031-025924x](https://github.com/CarlosEsco/Neko/assets/48522963/d3e80d68-a78c-4036-82d4-c811f5906042)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/docs/guides/troubleshooting/).
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
I have a MR for this on the library, once that is merged and release I can update Neko to have the fix
